bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -36,7 +36,7 @@ trait Asterism {
     * only possibilities.
     * TODO: should we Church encode this instead?
     */
-  def targets: Target \/ (SiderealTarget, SiderealTarget)
+  def targets: Target \/ (Target, Target)
```

Comment:
But *in practice* every asterism will be `Target \/ (SiderealTarget, SiderealTarget)` … anything else is an error condition that the UI should have prevented. It's irritating that we can't count on this in a stronger way. I don't know how to solve it other than getting rid of `SPTarget`.